src/main/topLevelPages/myDashboard/newVirtualContributorWizard/AddContent.tsx

```diff
@@ -40,7 +40,11 @@ const AddContent = ({ onClose, onCreateVC }: AddContentProps) => {
   const validationSchema = yup.object().shape({
     posts: yup.array().of(
       yup.object().shape({
-        title: yup.string().min(3, MessageWithPayload('forms.validations.minLength')).required(),
+        title: yup
+          .string()
+          .min(3, MessageWithPayload('forms.validations.minLength'))
+          .max(SMALL_TEXT_LENGTH, MessageWithPayload('forms.validations.maxLength'))
+          .required(),
```

Comment:
Can you also add a custom message for the title?

![image](https://github.com/user-attachments/assets/d2e2aebb-0c7e-4ee0-9631-69878fbf6fac)